### PR TITLE
[new release] textmate-language (0.3.1)

### PR DESCRIPTION
**Pull request by alan-j-hu**
Tokenizing code with TextMate grammars for syntax highlighting

- Project page: <a href="https://github.com/alan-j-hu/ocaml-textmate-language">https://github.com/alan-j-hu/ocaml-textmate-language</a>
- Documentation: <a href="https://alan-j-hu.github.io/ocaml-textmate-language/">https://alan-j-hu.github.io/ocaml-textmate-language/</a>

##### CHANGES:

- Add `find_by_filetype` function.

### CODE REVIEW

File: packages/textmate-language/textmate-language.0.3.1/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/alan-j-hu/ocaml-textmate-language.git"
```
Comment (mseri) [inline packages/textmate-language/textmate-language.0.3.1/opam:38]:
```suggestion
dev-repo: "git+https://github.com/alan-j-hu/ocaml-textmate-language.git"
x-ci-accept-failures: [
  "centos-7"
  "oraclelinux-7"
  "oraclelinux-8"
]
```

Comment (alan-j-hu) [reply packages/textmate-language/textmate-language.0.3.1/opam:38]: Is it better to add those restrictions to https://github.com/ocaml/opam-repository/blob/master/packages/conf-oniguruma/conf-oniguruma.1/opam?

**Comment by alan-j-hu** [comment]
Why is the CI still running the tests for the exempted distributions and reporting failures? It accepts failures on `oraclelinux-7` but not anything else?

In the case of packages not being found, should the restrictions be added to the `depexts` of https://github.com/ocaml/opam-repository/blob/master/packages/conf-oniguruma/conf-oniguruma.1/opam?

**Comment by mseri** [comment]
Oh, yes, sorry, my bad. If the failure is in the `conf-` package, my suggestion will not make any difference indeed.

**Comment by mseri** [comment]
Can you remove aa8bcbf? After that I will merge. Sorry again

**Comment by mseri** [comment]
Thanks a lot

**Pull request merged**